tests/core/pyspec/eth2spec/test/phase0/fork_choice/test_get_head.py

```diff
@@ -86,15 +86,16 @@ def test_chain_no_attestations(spec, state):
     check_head_against_root(spec, store, anchor_root)
     output_head_check(spec, store, test_steps)
 
+
     # On receiving a block of `GENESIS_SLOT + 1` slot
     block_1 = build_empty_block_for_next_slot(spec, state)
-    signed_block_1 = state_transition_and_sign_block(spec, state, block_1)
+    signed_block_1 = state_transition_and_sign_block(spec, state.copy(), block_1)
```

Comment:
These state copies are so that forkchoice has the right state in `execution_payload_states` as we treat all blocks as full, when we test pre-gloas tests. @fradamt suggested that a better approach would be to treat them all as empty and simply change the bid to some random hash. I haven't tested this approach yet.